packages/tabs/src/component-tabs.tsx

```diff
@@ -6,31 +6,31 @@ import React, {
   Children,
 } from 'react';
 import { classNames as cn } from '@chbphone55/classnames';
-import { tabs as c } from '@warp-ds/component-classes';
+import { gridLayout, tabs as ccTabs } from '@warp-ds/component-classes';
 import { debounce } from './utils';
 import type { TabsProps } from './props';
 
 const setup = (
   { className, contained, children, onClick, active, ...rest }: any,
   tabsRef,
-  wunderbarRef,
+  wunderbarRef
 ) => ({
   nav: cn({
     [className]: !!className,
-    [contained ? c.wrapperContained : c.wrapperUnderlined]: true,
+    [contained ? ccTabs.wrapperContained : ccTabs.wrapperUnderlined]: true,
```

Comment:
I guess `contained` doesn't matter here anymore?

```suggestion
    [ccTabs.wrapperUnderlined]: true,
```